### DISCUSSION TITLE
Reupload same photo?

### DISCUSSION THREAD

**Discussion by Godd67** [immich-app/immich#9413] [Q&A]
Here is the case:
- my photos are backed up immediately after taken
- then I edited some pictures - cropped etc.  and saved them (under the same name)
- Immich ignores them as it thinks they already been uploaded. The only way to get those photos to Immich is to save them under different name and then delete originals from Immich server.

Is it a way to reupload an already uploaded file manually?

**Comment by mmomjian** [comment]
You would have to reupload it to the web. The app does not currently have the ability to detect edits.
